src/main/resources/templates/city_info.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html xmlns:th="www.thymeleaf.org">
+<head>
+    <title>City</title>
+</head>
+<body> 
+    <div class="body">
+        <h2>City</h2>
+        <table>
+            <tr>
+                <th>ID</th>
+                <th th:text="${cityInfo.ID}"></th>
+            </tr>
+            <tr>
+                <th>Name</th>
+                <th th:text="${cityInfo.name}"></th>
+            </tr>
+            <tr>
+                <th>Country Code</th>
+                <th th:text="${cityInfo.countryCode}"></th>
+            </tr>
+            <tr>
+                <th>Country Name</th>
+                <th th:text="${cityInfo.countryName}"></th>
+            </tr>
+            <tr>
+                <th>District</th>
+                <th th:text="${cityInfo.district}"></th>
+            </tr>
+            <tr>
+                <th>Population</th>
+                <th th:text="${cityInfo.population}"></th>
+            </tr>
+            <tr>
+                <th>Local Weather</th>
+                <th th:text="${cityInfo.temp} + &#8457"></th>
+            </tr>
+        </table>
+        <p> For a fabulous weekend get away for 2 to 
+            <span th:text="${cityInfo.name}"></span> sign up here. </p>
+		<form th:action="@{/cities/reservation}" method="POST">
+		  <input type="hidden" id="city" name="city" th:value="${cityInfo.name}">
+		  <label for="email">Email: </label><br>
```

Comment:
Has the additional form for posting to the cities/reservation endpoint.  Looks like it will correctly post the city name, budget level, and email.